plugin-server/src/main/graphile-worker/worker-setup.ts

```diff
@@ -66,6 +72,7 @@ export function getPluginJobHandlers(hub: Hub, graphileWorker: GraphileWorker, p
     const pluginJobHandlers: TaskList = {
         pluginJob: async (job) => {
             pauseQueueIfWorkerFull(() => graphileWorker.pause(), hub, piscina)
+            jobsTriggeredCounter.inc()
```

Comment:
non-blocking as it's an improvement rather than straight lift and shift

Should we add in a success/failure after we're executed?

Reply:
Good idea, will do!